app/src/main/java/org/schabi/newpipe/fragments/list/playlist/PlaylistFragment.java

```diff
@@ -282,10 +284,21 @@ public void handleResult(@NonNull final PlaylistInfo result) {
 
         playlistCtrl.setVisibility(View.VISIBLE);
 
-        imageLoader.displayImage(result.getUploaderAvatarUrl(), headerUploaderAvatar,
-                ImageDisplayConstants.DISPLAY_AVATAR_OPTIONS);
-        headerStreamCount.setText(getResources().getQuantityString(R.plurals.videos,
-                (int) result.getStreamCount(), (int) result.getStreamCount()));
+        String avatarUrl = result.getUploaderAvatarUrl();
+        if (avatarUrl.contains("upload.wikimedia.org")) {
```

Comment:
I think checking whether the uploader name is "YouTube" and the uploaderUrl is better here

Reply:
```suggestion
        if (result instanceof YoutubeMixPlaylistExtractor) {
```

Reply:
```suggestion
        if (YouTubeParsingHelper.isYouTubeMixId(result.getId())) {
```

Reply:
That's not a good idea, unless you first check whether `result.getService()` equals YouTube's service idea.

Reply:
@wb9688 is `result.getServiceId() == ServiceList.YouTube.getServiceId()` ok?

Reply:
`result.getServiceId() == ServiceList.YouTube.getServiceId() && YouTubeParsingHelper.isYouTubeMixId(result.getId())` is OK.